include/IECorePython/RefCountedBinding.h

```diff
@@ -63,6 +63,27 @@ class RefCountedWrapper : public T, public WrapperGarbageCollector
 		template<typename Arg1, typename Arg2, typename Arg3>
 		RefCountedWrapper( PyObject *self, Arg1 arg1, Arg2 arg2, Arg3 arg3 );
 
+		template<typename Arg1, typename Arg2, typename Arg3, typename Arg4>
```

Comment:
I believe that with C++11 we can do this properly with a single variadic template. Seems a pity to do it the hard way when we're about to make the switch...